.github/workflows/snapshot_release.yml

```diff
@@ -35,7 +50,12 @@ jobs:
     steps:
       - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # version 4.1.4
       - uses: ./.github/actions/setup_node
+        with:
+          node-version: 18
       - uses: ./.github/actions/restore_build_cache
+        with:
+          node-version: 18
+          cdk-version: FROM_PACKAGE_LOCK
```

Comment:
where node-version is hard coded, we should extract to single place if possible with relevant name, maybe 'current supported node version'

Reply:
Can you see any alternative?

Reply:
If this introduces more complexity (more than what's already here), I don't see the value add for doing this, i.e. trying to extract default node version in a common place. Given that we would change this once every couple years?

Reply:
I was thinking environment variable set in workflow, but thats fine if not possible

Reply:
this was my thought if we were to set at the project level https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-configuration-variables-for-multiple-workflows

Reply:
Yaml directives are processed before job is ran. Env vars are not available at this stage - they can be used inside jobs but not in their declarations.

Reply:
> https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-configuration-variables-for-multiple-workflows

the `context vars` (not env vars) could work. but that's another trade-off - they would be outside of yml files in repo settings - not discoverable.